app/controllers/api/labwares_controller.rb

```diff
@@ -24,4 +28,11 @@ def labwares_by_location
 
     Labware.joins(:location).where(locations: { barcode: location_barcodes.split(',') })
   end
+
+  def labwares_by_barcode
+    barcodes = params[:barcodes]
+    return unless barcodes
+
```

Comment:
I'd suggest either:
  - returning an empty array here instead; so that the structure of the response remains the same
  - using params.require(:barcodes) instead, which will result in a 400 if the parameter is missing, letting the client know they've done something wrong. (I'm not sure what the actual response payload looks like in this case, so it may not be hugely useful by default)